docs/sdk-keys/target-apps.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Target Apps
+sidebar_label: Target Apps
+---
+
+Coming Soon!
+
+## Motivation
+
+There are two main attributes you can add to SDK Keys which restrict their access to certain configs or config definitions - Environments, and Target Apps.  A “Target App” is a user defined abstraction tied to entities in your Statsig Project which can be linked to one or more SDK Keys.
+Target apps should be things like your "Android" or "iOS" apps, or could be more restricted to specific services like "Search" and "Feed" which may span both client and server usage.
+
+Defining and using Target Apps is a best practice when using statsig, and has two main benefits:
+- Performance: removing unused entities from the payload to your statsig SDK integration will speed up the initialization time and reduce the amount of data that is stored in any local caches or data stores
+- Security: you may not want to expose certain gates/experiments/configs to the client side, or to the client key which is easily found in your client app. Specifying a target app for your client key and ONLY linking that to client specific gates/configs/experiments hides those from prying eyes
+
+:::info
+NOTE: Before using target apps as a solution, consider if you can instead use separate projects for your statsig integrations.  Note that gates/experiments/metrics are restricted to a single Statsig project, so if you need to share any of those across apps or services, its best to use a Target App instead. But if your services or apps always have distinct gates/experiments/metrics, using a separate project might be a better solution.
+:::
+
+If you have an existing Statsig project with multiple SDK integrations, Statsig can help you bootstrap your Target Apps by tagging existing entities which have recorded checks from that specific SDK.  This would help you at least segment your project by which sdks are accessing them
+
+## Example
+
+### Project Setup
+
+Start by going to Project Settings > Keys and Environments.  There you will find a "Target Apps" section.
+
+![Screen Shot 2023-07-26 at 4 18 46 PM](https://github.com/statsig-io/js-client/assets/74584483/6a2d9fa5-b3f9-4fd3-91e5-3487ebd02618)
+
+Create a target app, or use one that is suggested based on your current SDK exposure logs.
+
+![Screen Shot 2023-07-26 at 5 17 53 PM](https://github.com/statsig-io/docs/assets/74584483/719a399d-66f2-48a5-bf42-20f383758c7f)
+
+### Associating a Feature Gate with a Target App
+
+If you wish to verify the suggestions you added in the initial setup, or add additional entities to the target app, you can do so from that entity itself.
+
+For example, navigate to the Feature Gates tab, and then select a specific gate.
+
+In the header for that gate, you view and edit the target app(s)
```

Comment:
"In the header for that gate, you can view and edit the target app(s) applied to that gate."